prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -938,7 +937,7 @@ fi_addr_t rxr_pkt_insert_addr(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry
 			"Host %s received a packet with invalid protocol version %d.\n"
 			"This host can only support protocol version %d and above.\n",
 			host_gid, base_hdr->version, RXR_PROTOCOL_VERSION);
-		efa_eq_write_error(&ep->util_ep, FI_EIO, -FI_EINVAL);
```

Comment:
Was the negative code a bug?

Reply:
Yeah we want the `prov_errno` argument to `efa_eq_write_error()` to be a positive EFA-specific error code. I think we were just passing negative error codes as a placeholder.